Package.swift

```diff
@@ -17,16 +17,16 @@ let package = Package(
         .package(url: "https://github.com/vapor/vapor.git", from: "4.0.1"),
         .package(url: "https://github.com/vapor/fluent.git", from: "4.0.0-rc.2"),
         .package(url: "https://github.com/vapor/sql-kit.git", from: "3.0.0-rc"),
-        .package(url: "https://github.com/vapor/queues.git", from: "1.0.0"),
+        .package(url: "https://github.com/vapor/queues.git", from: "1.0.0-rc"),
```

Comment:
Was hoping Queues would go out of RC a few days ago, hasn't happened yet. Let's switch back to `-rc` for now.

Reply:
Ok